docs/concepts/service-catalog/index.md

```diff
@@ -95,7 +95,7 @@ The following is a sequence diagram illustrating the steps involved in listing M
 
 1. Once the `ClusterServiceBroker` resource is added to Service Catalog, it triggers a *List Services* call to the external Service Broker.
 1. The Service Broker returns a list of available Managed Services and Service Plans, which are cached locally in `ClusterServiceClass` and `ClusterServicePlan` resources.
```

Comment:
It's the list that is cached, correct? It's not the Services and ServicePlans that are cached.

So I think this should be "which is cached locally".

Reply:
Yes, the lists are cached in the local `ClusterServiceClass` and `ClusterServicePlan` resources. How about "The service broker returns a list of available managed services and a list of Service Plans, which are cached locally as `ClusterServiceClass` and `ClusterServicePlan` resources respectively."